code/datums/spells/turf_teleport.dm

```diff
@@ -1,14 +1,19 @@
 /obj/effect/proc_holder/spell/targeted/turf_teleport
 	name = "Turf Teleport"
 	desc = "This spell teleports the target to the turf in range."
+	nonabstract_req = 1
 
 	var/inner_tele_radius = 1
 	var/outer_tele_radius = 2
 
 	var/include_space = 0 //whether it includes space tiles in possible teleport locations
 	var/include_dense = 0 //whether it includes dense tiles in possible teleport locations
+	
+	var/sound1 = "sound/weapons/ZapBang.ogg"
+	var/sound2 = "sound/weapons/ZapBang.ogg"

```

Comment:
These are filename strings, but they should be (single-quoted) resource identifiers, so the sounds get properly cached (and sanity checked) during compilation. There seems to be quite a few of these scattered throughout this PR... and elsewhere in our codebase.